app/components/Map.js

```diff
@@ -103,7 +146,8 @@ export default class Map extends Component {
 
 
   render() {
-    const { pins, getLocationToSave, currLoc, recent } = this.props;
+    // TODO: Map is re-rendering continually. Fix bug
```

Comment:
When you're standing still the map doesn't seem to be re-rendering. I do see a performance hit when testing this PR though.